### PR TITLE
A couple of docs tweaks for fdot

### PR DESCRIPTION
**Pull request by landonreed**
This PR:
- Removes the unused about page
- Removes the unused Terms & Conditions page
- Changes the Help page to a mailto link
- Adds more helpful info about API keys and how to request one with fewer limits on requests.

### CODE REVIEW

**Review by evansiroky — APPROVED** [review]

File: components/Footer.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import NavLink from './NavLink'
 export default function Footer (props) {
```
Comment (binh-dam-ibigroup) [inline components/Footer.js:9]: Need to remove the "About" link too (see https://otp-admin-ui-git-fdot-tweaks-ibi-group.vercel.app/about)

File: components/ApiUserDashboard.js [hunk 2]
```diff
@@ -34,9 +35,15 @@ class ApiUserDashboard extends Component {
...
-          Note: copy/paste the API key values below for use in
-          the <code>x-api-key</code> header when making any request to the RMCE
-          API.
+          <Alert.Heading>
+            API Key Instructions
+          </Alert.Heading>
+          <p>
+            Below you will find your API keys to make requests to the trip planner.
+            These keys should only be used for testing because the request limit
+            is heavily restricted.
```
Comment (binh-dam-ibigroup) [inline components/ApiUserDashboard.js:44]: Is this permanent text?

File: components/ApiUserDashboard.js [hunk 3]
```diff
@@ -34,9 +35,15 @@ class ApiUserDashboard extends Component {
...
+          </p>
+          <EmailForApiKeyMessage />
```
Comment (binh-dam-ibigroup) [inline components/ApiUserDashboard.js:46]: Not sure why this is a separate component (it puts the above text regarding limits out of context).

File: components/Footer.js [hunk 4]
```diff
@@ -75,29 +75,16 @@ export default function Footer (props) {
...
-            </li>
           </ul>
         </div>
       </section>
       <section className='ft-legal'>
         <ul className='ft-legal-list'>
-          <li>
-            <NavLink
-              href={termsPath}>
-              Terms &amp; Conditions
```
Comment (binh-dam-ibigroup) [inline components/Footer.js:98]: A terms an conditions link is shown in the user sign up page, is that still the intent?

File: components/ApiUserDashboard.js (hunk 3 above)
Comment (binh-dam-ibigroup) [reply components/ApiUserDashboard.js:46]: I'd suggest to inline that content, or place a comment saying why it shouldn't be inlined.

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Please remove the broken "About" link and advise regarding whether to keep the link to T&Cs in the user sign up page.

File: components/Footer.js (hunk 4 above)
Comment (landonreed) [reply components/Footer.js:98]: Nope, good catch!

Comment (landonreed) [reply components/Footer.js:98]: Fixed in c21b67c

File: components/Footer.js (hunk 1 above)
Comment (landonreed) [reply components/Footer.js:9]: Fixed in c21b67c

**Review by binh-dam-ibigroup — APPROVED** [review]
Looks good now, thanks for the changes!

**Pull request merged**